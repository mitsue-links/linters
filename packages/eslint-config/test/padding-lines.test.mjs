import path from 'path';
import {lintFile} from './helper.mjs';

test('padding-lines_ok.mjs', async () => {
    const filePath = 'padding-lines_ok.mjs';
    const data = await lintFile(path.join('fixtures', filePath));

    expect(data).toStrictEqual([{
        filePath,
        messages: [],
    }]);
});

test('padding-lines_error.mjs', async () => {
    const filePath = 'padding-lines_error.mjs';
    const data = await lintFile(path.join('fixtures', filePath));

    expect(data).toMatchSnapshot();
});
