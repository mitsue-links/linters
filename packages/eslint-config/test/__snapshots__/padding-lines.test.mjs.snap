// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`padding-lines_error.mjs 1`] = `
[
  {
    "filePath": "padding-lines_error.mjs",
    "messages": [
      {
        "line": 13,
        "ruleId": "vars-on-top",
        "severity": 2,
      },
      {
        "line": 17,
        "ruleId": "vars-on-top",
        "severity": 2,
      },
    ],
  },
]
`;
